juriscraper/opinions/united_states/state/calctapp_1st.py

```diff
@@ -11,7 +11,3 @@ def __init__(self, *args, **kwargs):
         self.court_id = self.__module__
         self.court_code = "A"
         self.division = "1st App. Dist."
-        self.url = self.build_url()
```

Comment:
Here you need to rebuild the url to update the court_code because it is generated in parent __init__ and not being updated when you override it in calctapp_1st

`self.url = f"http://www.courts.ca.gov/cms/opinions.htm?Courts={self.court_code}"`

otherwise it will always get the supreme court cases

`python sample_caller.py -c juriscraper.opinions.united_states.state.calctapp_1st -vv`